README.rst

```diff
@@ -24,7 +24,14 @@ RNASeq normalization
     :target: https://pepy.tech/project/rnanorm
     :alt: Number of downloads from PyPI
 
-Normalization of RNA-seq gene expression. Supported methods: Transcript per kilobase million (TPM) and Counts per million (CPM).
+Normalization of RNA-seq gene expression data. Supported methods are Transcript
+per kilobase million (TPM) and Counts per million (CPM).
+
+The TPM normalization can either accept pre-computed gene lengths on the input
+or compute gene lengths from gene annotation in GTF format, using the union
+exon-based approach. The computed gene lengths are identical to the lengths
```

Comment:
The pre-computed gene lengths are identical ...

Reply:
With pre-computed I refer to the scenario when you feed the tool a list of pre-computed gene lengths.

In this context I could use: "The on-the-fly-computed gene lengths are identical ..." But "on-the-fly-computed" sounds a bit awkward, any suggestions?